src/Python/logger.py

```diff
@@ -1,37 +1,68 @@
 # Read and Store MPU6050 Data to Disk
-
+import csv
 import serial
 from serial.serialutil import SerialException
 
 import time
 import argparse
+import os
+
+# Create timestamp to log entries and create log file
+date_str = time.strftime("%Y-%m-%d_%H-%M-%S", time.localtime())
 
 # Parse command line arguments
 parser = argparse.ArgumentParser()
 parser.add_argument("--port", type=str, default="/dev/ttyACM0")
 parser.add_argument("--baud_rate", type=int, default=115200)
+parser.add_argument("--log_dir", type=str, default="./logs")
+parser.add_argument("--csv_file_name", type=str, default=f"imu_data_{date_str}.csv")
 args = parser.parse_args()
 
+# Create log directory if it does not exist
```

Comment:
Can ensure the log directory exists. Create empty logs/ and place a .gitignore inside of it with
```
*
!.gitignore
```